integration-tests/tests/api/schema/external-composition.spec.ts

```diff
@@ -81,3 +81,93 @@ test.concurrent('call an external service to compose and validate services', asy
   // expect `products` service to be composed externally
   await expect(history()).resolves.toContainEqual(productsServiceName);
 });
+
+test.concurrent(
```

Comment:
@kamilkisiela can we please also have a test that tried to call a service that does not exists?  just to make sure the `fetch` network error is also visible to the user

Reply:
that's exactly what I'm doing now